docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3'
+services:
+  web:
+    container_name: forge-web
+    build: .docker/nginx/
+    volumes:
+      - .:/home/forge-hr/
+    labels:
+    - "traefik.backend=forge-api"
+    - "traefik.frontend.rule=Host:api.forge.test"
+    - "trafik.enable=true"
+    - "traefik.port=80"
+    - "traefik.protocol=http"
+    - "traefik.frontend.entryPoints=http"
+    networks:
+      - traefik
+  db:
+    container_name: forge-db
+    image: postgres:10
+    ports:
+      - 55432:5432
+    volumes:
+      - ./.docker/.data/db:/var/lib/postgresql
+    environment:
+      POSTGRES_USER: swdht
+      POSTGRES_PASSWORD: swdht
+      POSTGRES_DB: swdht
+    networks:
+      - traefik
+  php:
+    container_name: forge-php
+    build: .docker/php/
+    volumes:
+      - ./api/:/home/forge-hr/api
+    networks:
+      - traefik
+
+  composer:
+    image: 'composer:latest'
+    volumes:
+      - ./api/:/home/forge-hr/api
+  front:
+    container_name: forge-front
+    build: .docker/node
+    working_dir: /home/forge-hr/front
+    user: "node"
+    volumes:
+      - ./front/:/home/forge-hr/front
+    labels:
+    - "traefik.backend=forge-front"
+    - "traefik.frontend.rule=Host:forge.test"
+    - "trafik.enable=true"
+    - "traefik.port=8080"
+    - "traefik.protocol=http"
+    - "traefik.frontend.entryPoints=http"
+    networks:
+      - traefik
+
+  traefik:
+    image: traefik
+    command: --web --docker --docker.domain=docker.localhost --logLevel=DEBUG
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    labels:
+      - "traefik.frontend.rule=Host:srv.forge.test"
+      - "traefik.backend=traefik"
+      - "traefik.port=8080"
+    networks:
+      - traefik
+
+networks:
+   traefik:
+     external: true
```

Comment:
You cannot expose the service in an external network by default, because all developers won't have a global traefik instance on their laptop. Instead, override locally your `docker-compose.yaml` to expose all services into an external network. By default, it must be in the default-provided network, i.e "bridge" and internal.